client/app/components/action-form-input.js

```diff
@@ -0,0 +1,23 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class ActionFormInputComponent extends Component {
+  // represents pasForm model
+  @tracked pasForm;
+
+  // attribute on pasForm model, e.g. 'dcpPfchangeincitymap'
+  @tracked landUseActionField;
```

Comment:
It looks like the 3 arguments for this compnent are : 
`pasForm, fieldType, landUseActionField`

Those 3 don't need to be tracked. If you want to provide default values, see comment above :)